### PR TITLE
Compose files for specific robots

### PR DESCRIPTION
**Pull request by AndrewCouncil**
Created docker compose file for specific robots and a line to the entrypoint to export an environment variable from the compose file

### CODE REVIEW

File: docker-compose-cthulhu.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+version: "3.3"
+
+services:
+  onboard-cthulhu:
+    image: dukerobotics/robosub-ros:onboard
+    container_name: onboard-cthulhu
+    environment:
+      - ROBOT_NAME=cthulhu
```
Comment (ShaanGondalia) [inline docker-compose-cthulhu.yml:8]: Is the only difference in these compose files the existence of these environment variables? If so is there some other way to inject the robot name into the container without needing duplicate compose files? Maybe we can refactor our code to use [Extension](https://docs.docker.com/compose/extends/#extending-services).

Comment (Brokemia) [reply docker-compose-cthulhu.yml:9]: Made that change

**Comment by AndrewCouncil** [comment]
this has now been tested and is working. for any shell open under root user the environment variable ROBOT_NAME will be set to the name of the robot. can be checked in python using [os.environ](https://docs.python.org/3/library/os.html#os.environ)

**Review by Brokemia — APPROVED** [review]
Looks good

**Pull request merged**